test_analysis5.py

```
@@ -0,0 +1,61 @@
+            tlines = f.readlines()
+            tcount = len(tlines)
+        print(f"HAS TEST ({tcount} lines): {t} ({dict(sizes)[t]} lines)")
+    else:
+        # check alternative paths
+        name = os.path.basename(t).replace('.dart', '')
+        found = False
+        for root, _, files in os.walk('test'):
+            for file in files:
+                if name in file:
+                    test_file = os.path.join(root, file)
+                    with open(test_file, 'r') as f:
```

Ruff (0.15.13)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)